tests/unit/scm/test_scm.py

```diff
@@ -37,3 +37,9 @@ def test_should_throw_and_cleanup(self):
 
         self.assertEqual(0, self.scm_mock.reset_ignores.call_count)
         self.assertEqual(0, self.scm_mock.remind_to_track.call_count)
+
+
+def test_remind_to_track(scm, caplog):
+    scm.files_to_track = ["fname with spaces.txt", "тест", "foo"]
+    scm.remind_to_track()
+    assert "git add 'fname with spaces.txt' 'тест' foo" in caplog.text
```

Comment:
I remember windows cmdline being weird about single-quotes and having to use double-quotes. Also IIRC windows cmd always does completion with double-quotes. Searching around (e.g. https://ss64.com/nt/syntax-esc.html) I also see double quotes used and not single-quotes. Let's switch to double-quotes too, it will be compatible with both win and *nix.

Reply:
well *nix will shell-expand things in double quotes:

```bash
$ export foo=bar
$ echo spam > \$foo
$ cat "$foo"  # double quotes :(
cat: bar: No such file or directory
```

Reply:
Also @efiop this PR uses `shlex.quote` which avoids this

Reply:
@casperdcl Yep, there is also `mslex` which properly quotes things for windows, but it doesn't have a conda package. I'm wondering if it is easier to just make dvc automatically `git add` things at this point. Easier for both us on implementation and for users in terms of UX.

Reply:
yes not sure if I've mentioned this somewhere before but simply adding `--git` to any `dvc` command to auto-exec the obvious would be nice